main.py

```diff
@@ -12,7 +12,7 @@
 logger = logging.getLogger(__name__)
 fake = Faker()
 
-MAILINATOR_URL = "https://www.mailinator.com/v2/inbox.jsp?zone=public&query="
+MAILINATOR_URL = "https://mailinator.com/v3/index.jsp?zone=public&query="
```

Comment:
I try this for example and return a redirect page.
https://www.mailinator.com/v3/inbox.jsp?zone=public&query=asdmpowell1070

Reply:
Its v3/index.jsp why are u trying v3/inbox.jsp?
Sorry, if u r just commenting additional info.

Reply:
So sorry, my url was bad.
Thank you!.